engine/src/test/java/org/camunda/bpm/engine/test/persistence/ConnectionPersistenceExceptionTest.java

```diff
@@ -38,7 +38,8 @@
 import org.junit.Rule;
 import org.junit.Test;
 
-@RequiredDatabase(excludes = { DbSqlSessionFactory.H2 })
+// This test is excluded on Oracle since the SQL State changed with the new version of the JDBC driver.
+@RequiredDatabase(excludes = { DbSqlSessionFactory.H2, DbSqlSessionFactory.ORACLE })
```

Comment:
❓ Don't we need to handle the new code in the implementation instead of excluding the db for this test?

Reply:
The feature aims to react to the `08` class for SQL states. These states still exist with the new JDBC driver, but the test no longer triggers this. I have no idea how to provoke this for Oracle. The customer who requested the feature needs it for PostgreSQL anyways. I doubt the time is well invested to find another way to provoke the class `08` SQL state on Oracle under these circumstances. However, I'm open for ideas. I think excluding the test is a pragmatic approach.

Reply:
Ok, I agree. Marking as resolved.